### DISCUSSION TITLE
Issue with multi-GPU support in Auto3DSeg on Windows

### DISCUSSION THREAD

**Discussion by Yasna-frn** [Project-MONAI/tutorials#1701] [General]
When running Auto3DSeg on Windows, I encountered the error. However, when I utilized %env CUDA_VISIBLE_DEVICES to target a single GPU, the program executed without any issues. It appears that there might be a problem specifically related to multi-GPU usage within Auto3DSeg on the Windows platform. Is there any solution to fix it?
![Screenshot 2024-04-23 151120](https://github.com/Project-MONAI/tutorials/assets/145801636/344ddc75-c22e-4f18-a49c-a69e4804f77c)

**Comment by KumoLiu** [comment]
Hi @Yasna-frn, thanks for your interest here. The current version of Auto3DSeg algorithm use a Linux library and can't be used on Windows at this point.
https://github.com/Project-MONAI/MONAI/issues/6821#issuecomment-1789933361
